src/pages/about.tsx

```diff
@@ -1,12 +1,49 @@
-import { Typography } from "@mui/material";
+import { Stack, Typography } from "@mui/material";
 
 import { text } from "~/assets/text/index.js";
+import { styles } from "~/styles/pages/about.js";
+
+const wordsToLink: Record<string, string> = {
+  "Kryha Labs": "https://kryha.io",
+  Aleo: "https://aleo.org",
+  zPass: "https://zpass.aleo.org/",
+};
 
 function About() {
+  const getContent = () => {
```

Comment:
You could abstract some of these utilities that map and check for link outside of the page file.